frontend/components/Chain.vue

```diff
@@ -256,22 +256,22 @@ export default {
         `,
         result({ data }) {
           this.totalExtrinsics =
-            data.total.find((row) => row.name === 'extrinsics').count || 0
+            data.total.find((row) => row.name === 'extrinsics')?.count || 0
```

Comment:
Why do you need it?

Reply:
It fixed a "field name is undefined" error.

Reply:
Why it was not an issue previously but now become?

Reply:
Because now I tested a clean Cere node version with empty data. Before this data was not empty.